src/routes/projects.rs

```diff
@@ -238,6 +249,22 @@ pub struct EditProject {
     )]
     #[validate(length(min = 3, max = 64), regex = "RE_URL_SAFE")]
     pub slug: Option<Option<String>>,
+
+    pub status: Option<ProjectStatus>,
+    #[serde(
+        default,
+        skip_serializing_if = "Option::is_none",
+        with = "::serde_with::rust::double_option"
+    )]
+    #[validate(length(max = 2000))]
+    pub rejection_reason: Option<Option<String>>,
```

Comment:
Double option :concern:

Reply:
Double options are used to remove the rejection reason. Speaking of which, I should make it so when the user sends the mod for approval the rejection reason is removed